cmd/appgw-ingress/main.go

```diff
@@ -45,11 +45,17 @@ var (
 func main() {
 	flags.Parse(os.Args)
 
+	envVariables := newEnvVariables()
```

Comment:
In the future we could also move this entire block in `newEnvVariables` and use [os.LookupEnv](https://golang.org/pkg/os/#LookupEnv) to combine the getting and checking of the env variable.